packages/composedb/composites/1-socialHandle.graphql

```diff
@@ -0,0 +1,10 @@
+type SocialHandle
+  @createModel(accountRelation: LIST, description: "A social handle on other platforms")
+  @createIndex(fields: [{ path: "platform"}])
+  @createIndex(fields: [{ path: "handle"}])
+{
+  owner: DID! @documentAccount
+  version: CommitID! @documentVersion
+  platform: String! @string(maxLength: 256)
+  handle: String! @string(maxLength: 256)
+}
```

Comment:
@hubsmoke basically just this that would be nice with some eyes on. I don't love `platform` because it doesn't fit protocols (email, for example). I also don't like `SocialHandle` as a name for the same reasons :monocle_face: